openmc/data/njoy.py

```diff
@@ -127,7 +127,7 @@
 1 0 1 .{ext} /
 '{library}: {zsymam} at {temperature}'/
 {mat} {temperature}
-1 1/
+1 1 {ismoothing}/
```

Comment:
Just to be consistent with the NJOY manual
```suggestion
1 1 {ismooth}/
```